### PR TITLE
Prep Javascript Variables - Anthony Garcia

### PR DESCRIPTION
**Pull request by Garcia-Tony**
Screen Shot 2024-04-20 at 2 47 23 PM

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      console.log('False');
+      console.log(typeof likesVegetables);
+    </script>
+    <script>
+      const result = null;
+      console.log(result);
+      console.log(typeof result);
+    </script>
+    <script>
+      let = undefined;
```
Comment (BrettAlbright) [inline prep-javascript-variables/index.html]: make sure you add a `variable` here

let nothing = undefined

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
tiny update needed. 

Great work so far!

**Comment by Garcia-Tony** [comment]
Screen Shot 2024-04-22 at 8 14 14 PM
Screen Shot 2024-04-22 at 8 14 54 PM

**Review by BrettAlbright — APPROVED** [review]
this does work, but in the future, I would only use one `script` tag to hold all of the JS

**Pull request merged**